packages/lab/src/tabs/ActivationIndicator.css

```diff
@@ -1,22 +0,0 @@
-/* the Tabstrip Activation Indicator, uses navigable styles */
```

Comment:
Unused file?

Reply:
can delete, was moved to TabActivationIndicator